yarn.lock

```diff
@@ -1787,9 +1787,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001109, caniuse-lite@^1.0.30001219:
-  version "1.0.30001228"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001228.tgz#bfdc5942cd3326fa51ee0b42fbef4da9d492a7fa"
-  integrity sha512-QQmLOGJ3DEgokHbMSA8cj2a+geXqmnpyOFT0lhQV6P3/YOJvGDEwoedcwxEQ30gJIwIIunHIicunJ2rzK5gB2A==
+  version "1.0.30001442"
```

Comment:
Just checking this isn't accidental - assume it was needed to get your environment working again?

Reply:
Yes this was from the reinstalls - thought I would leave it there as it seemed like and updated version, but can remove if it causes problems..

Reply:
I'm sure it'll be alright - yes can always revert if need be later